### PR TITLE
Correct size action

### PR DESCRIPTION
**Pull request by ferhatelmas**

### CODE REVIEW

**Comment by ferhatelmas** [comment]
@mahboubii this is safe to merge because failure is that it can't find the new command in the base branch to compare.

File: bin/version.sh [hunk 1]
```diff
@@ -1,4 +1,3 @@
 set -e
-./node_modules/.bin/webpack
-./node_modules/.bin/webpack --minify
-npm run build
+yarn run dist
+yarn run build
```
Comment (mahboubii) [inline bin/version.sh:3]: let's remove this .sh file and just have this as in package.json scrips
like

File: package.json [hunk 2]
```diff
@@ -28,6 +28,7 @@
     "generate-docs": "jsdoc -c .jsdoc",
     "coverage": "nyc --reporter=text-lcov mocha --require @babel/register bin/run-node-unit-tests.js | codecov --pipe",
     "preversion": "yarn run test-unit-node",
+    "dist": "webpack && webpack --minify",
     "version": "./bin/version.sh && git add dist",
```
Comment (mahboubii) [inline package.json:32]:
```suggestion
    "version": "yarn dist && yarn build && git add dist",
```

File: bin/version.sh (hunk 1 above)
Comment (ferhatelmas) [reply bin/version.sh:3]: SGTM

**Pull request merged**